Makefile

```diff
@@ -8,9 +8,13 @@ pytest:
 check-trained-agents:
 	python -m pytest -v tests/test_enjoy.py -k trained_agent --color=yes
 
-# Type check
-type:
-	pytype -j auto rl_zoo3/ tests/ scripts/ -d import-error
+pytype:
+	pytype -j auto ${LINT_PATHS} -d import-error
+
+mypy:
+	mypy ${LINT_PATHS} --install-types --non-interactive
```

Comment:
Should we also add `--install-types --non-interactive` in sb3 and sb3-contrib?

Reply:
I would only if it was needed.